modules/howtos/examples/managing_connections.py

```diff
@@ -1,11 +1,20 @@
+# requires:
+#  - a local couchbase server with
+#      username/password
+#      travel-sample
+#      beer-sample
```

Comment:
eeek, need to get rid of these beer sample dependencies eventually 😓

Reply:
I kept that one in, as it's a *secondary* one, to prove the point that we *can* connect to more than one bucket.
Do we have a preferred secondary?

Reply:
🤔  So just thinking when we automate this, if the code needs to reference `beer-sample`, we may need to install it on Couchbase Server, but since `beer-sample` wasn't going to be used in our docs anymore not sure if we want to do that.  We could also just create a test bucket programmatically with the SDK, in the example that requires it, this way it keeps things isolated as not many examples will need a secondary bucket.

Reply:
that's fine. Essentially, all I've done is *document* it so that it's easy to tell what you need in order to run the code.

So fine to stick with beer-sample for now, and change it to a different one when we have an automation setup script.